src/sentry/api/endpoints/user_appearance.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import absolute_import
+
+from datetime import datetime
+
+import pytz
+from django.conf import settings
+from rest_framework.response import Response
+from rest_framework import serializers
+from django.utils.translation import ugettext_lazy as _
+
+from sentry.api.bases.user import UserEndpoint
+from sentry.constants import LANGUAGES
+from sentry.models import UserOption
+
+
+def _get_timezone_choices():
+    results = []
+    for tz in pytz.common_timezones:
+        now = datetime.now(pytz.timezone(tz))
+        offset = now.strftime('%z')
+        results.append((int(offset), tz, '(UTC%s) %s' % (offset, tz)))
+    results.sort()
+
+    for i in range(len(results)):
+        results[i] = results[i][1:]
+    return results
+
+
+TIMEZONE_CHOICES = _get_timezone_choices()
+
+
+class UserAppearanceSerializer(serializers.Serializer):
+    # Note the label part of these ChoiceFields are not used by the frontend
+    language = serializers.ChoiceField(choices=LANGUAGES, required=False)
+    stacktrace_order = serializers.ChoiceField(choices=(
```

Comment:
this and `clock_24_hours` should be camel case rather than snake case for consistency with the rest of our api (and in uses below)